students/Ruohan/lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python3
+
+import sys
+import string
+
+old_datas = [['William Gates III',  653784.39, 2],
```

Comment:
better to store all old donations, and compute sum when you need it. So if you wanted other info, you could get it.